software/control_sw/src/cosmic_fengine.py

```diff
@@ -905,6 +905,9 @@ def set_lo_fshift_list(self, lo_fshift_list, skip_if_equal=False):
             self.lo.set_target_load_time(int(lo_load_time * FPGA_CLOCK_RATE_HZ), enable_trig=True)
             self.logger.info(f"F-Shift load time set to {time.ctime(lo_load_time)}")
             time.sleep(lo_load_time - (time.time()))
+            fshift_time_to_load = self.lo.get_time_to_load()/FPGA_CLOCK_RATE_HZ
```

Comment:
A 1ms allowance which results in the code crashing if not met seems a little aggressive, no?

Reply:
maybe log an error but continue? Or at least make the test more relaxed (the tolerance on NTP vs board time is only ever tested to something like 10ms, I think)

Reply:
Probably is a little aggressive... it is what we use for the delay tracking see [here](https://github.com/realtimeradio/vla-dev/blob/phase_correct_loaded_fshifts/software/control_sw/src/cosmic_fengine.py#L1591).
Perhaps many of those asserts should be more lenient. Give it a look over an I'll make whatever changes you think necessary...

Reply:
Maybe go with 10ms and call it good. The main thing is, if you're going to raise exceptions, you absolutely _must_ notice and fix things if they inappropriately raise

Reply:
Loosened to 10ms